ddtrace/tracer/payload.go

```diff
@@ -32,10 +32,11 @@ type payload struct {
 	// off specifies the current read position on the header.
 	off int
 
-	// count specifies the number of items in the stream.
-	count uint64
+	// traces holds the sequence of traces to returns.
+	traces []spanList
+
+	s int //encoded payload size in bytes
```

Comment:
`s` is a bit ambiguous, how about `msgSize`? Also, do you mind putting the comment above so it matches the other fields?